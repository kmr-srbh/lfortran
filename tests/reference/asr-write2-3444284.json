{
    "basename": "asr-write2-3444284",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/write2.f90",
    "infile_hash": "e32f4a7448b2f59536da5f9579270c896bf9d80ebadd03daccc928a1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-write2-3444284.stdout",
    "stdout_hash": "ccf43a66cae9ffd80039b4dfb36db3d61b9cc10d67048f2f1b86b69d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}