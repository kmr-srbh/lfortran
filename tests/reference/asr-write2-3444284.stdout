(TranslationUnit
    (SymbolTable
        1
        {
            write2:
                (Program
                    (SymbolTable
                        2
                        {
                            i:
                                (Variable
                                    2
                                    i
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    write2
                    []
                    [(FileWrite
                        0
                        ()
                        ()
                        ()
                        ()
                        [(StringFormat
                            (StringConstant
                                "(a)"
                                (Character 1 3 () PointerString)
                            )
                            [(StringConstant
                                "Some text: "
                                (Character 1 11 () PointerString)
                            )]
                            FormatFortran
                            (Character -1 0 () PointerString)
                            ()
                        )]
                        ()
                        (StringConstant
                            ""
                            (Character 1 0 () PointerString)
                        )
                        ()
                    )
                    (FileRead
                        0
                        ()
                        ()
                        ()
                        ()
                        ()
                        ()
                        [(Var 2 i)]
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "Got: "
                                (Character 1 5 () PointerString)
                            )
                            (Var 2 i)]
                            FormatFortran
                            (Character -1 0 () PointerString)
                            ()
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
